articles/active-directory/develop/index-web-app.yml

```
@@ -38,7 +38,7 @@ landingContent:
         links:
           - text: ASP.NET
             url: tutorial-v2-asp-webapp.md
-          - text: Blazor Server
+          - text: Blazor WebAssembly
             url: tutorial-blazor-webassembly.md
```

Text was correct, target was incorrect. Should be:

```suggestion
          - text: Blazor Server
            url: tutorial-blazor-server.md
```